components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/index.html

```diff
@@ -1491,12 +1491,11 @@ <h4 class="modal-title file-dialog-title">Siddhi Store Query</h4>
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                     <i class="fw fw-cancel about-dialog-close"></i></button>
-                <h3 class="modal-title" id="aboutDockerExportLabel">Export as Docker</h3>
+                <h4 class="modal-title file-dialog-title" id="aboutDockerExportLabel">Save To Workspace</h4>
```

Comment:
Does this mean we don't have "Export as Docker" title anymore?

Reply:
Title text should be "Export as Docker"

Reply:
added the changes